kuma/javascript/src/components/a11y-nav.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow
+import * as React from 'react';
+
+import { gettext } from '../l10n.js';
+
+export default function A11yNav() {
+    return (
+        <ul id="nav-access">
+            <li>
+                <a id="skip-main" href="#content">
+                    {gettext('Skip to main content')}
+                </a>
+            </li>
+            <li>
+                <a id="skip-language" href="#language">
+                    {gettext('Select language')}
+                </a>
+            </li>
+            <li>
+                <a id="skip-search" href="#q">
+                    {gettext('Skip to search')}
+                </a>
+            </li>
+        </ul>
+    );
+}
```

Comment:
Perhaps it's nitpickery but I really don't like the location of this file. "components". 

I've seen projects that organize their files like this:

* "components"
  * "Home"
  * "About"
  * "Dashboard"
* "views"
  * "Home"
  * "About"
  * "Dashboard"
* "test"
  * "Home"
  * "About"
  * "Dashboard"

And I think that's a mistake because, if you decide to remove the "Dashboard" you have to search around for it all over the code base. A much better structure would be:

* "Home"
  * "component"
  * "view" 
  * "tests"
* "About"
  * "component"
  * "view" 
  * "tests"
* "Dashboard"
  * "component"
  * "view" 
  * "tests"

"Design for deletability" someone said and I can't find who. 

I'd much rather see a directory called "a11y" or something. Or, simply just call it "kuma/javascript/src/a11y-nav.jsx` which I think is a lot more matching to the kind of naming style that David started.